### PR TITLE
PAINTROID-583: Add PR number to apk file output

### PR DESCRIPTION
**Pull request by bhav-khurana**
## Ticket
[PAINTROID-583](https://catrobat.atlassian.net/browse/PAINTROID-583)

## New Features and Enhancements
- Adds PR number in the apk file name that is built during the GitHub actions.

#### Your checklist for this pull request
Please review the [contributing guidelines](https://github.com/Catrobat/Paintroid/blob/develop/README.md) and [wiki pages](https://github.com/Catrobat/Catroid/wiki/) of this repository.

- [x] Include the name of the Jira ticket in the PR’s title
- [x] Include a summary of the changes plus the relevant context
- [x] Choose the proper base branch (*develop*)
- [x] Confirm that the changes follow the project’s coding guidelines
- [x] Verify that the changes generate no compiler or linter warnings
- [x] Perform a self-review of the changes
- [x] Verify to commit no other files than the intentionally changed ones
- [ ] Include reasonable and readable tests verifying the added or changed behavior
- [x] Confirm that new and existing unit tests pass locally
- [x] Check that the commits’ message style matches the [project’s guideline](https://github.com/Catrobat/Catroid/wiki/Commit-Message-Guidelines)
- [x] Stick to the project’s gitflow workflow
- [x] Verify that your changes do not have any conflicts with the base branch
- [x] After the PR, verify that all CI checks have passed
- [ ] Post a message in the *#paintroid* [Slack channel](https://catrobat.slack.com) and ask for a code reviewer

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -24,10 +24,12 @@ jobs:
       - name: Widget Tests
         run: melos run test:widget
       - name: Build release package
-        run: flutter build apk --release
+        run: |
+          flutter build apk --release
+          mv build/app/outputs/flutter-apk/app-release.apk build/app/outputs/flutter-apk/app-release-${{ github.event.number }}.apk
```
Comment (msesko) [inline .github/workflows/main.yml]: do we need to rename the apk or do we just change the name parameter below? this gets deleted after a week so we are not creating storage that needs to be named properly right? this approach would require only changing the name parameter in line 33. @bakicelebi opinion?

Comment (bhav-khurana) [reply .github/workflows/main.yml]: Yeah, but when we download the apk we get a zip file containing `app-release.apk`. So I think we should rename that also to `app-release-{{PR number}}.apk` as we'll be extracting that only.
Is it so @juliajulie95 ma'am?

Comment (msesko) [reply .github/workflows/main.yml]: Yeah then it is better I think

**Comment by juliajulie95** [comment]
Could you please name it
flutter_paint_{PR-number}

I am switching between Paintroid native and flutter a lot and get them confused easily because the icon and the names are too similar :)

**Comment by bhav-khurana** [comment]
@juliajulie95 I've renamed it as required :)

**Review by msesko — APPROVED** [review]
Good

**Review by juliajulie95 — APPROVED** [review]

**Comment by juliajulie95** [comment]
Thank you!

**Pull request merged**